cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -231,6 +231,19 @@ def build_source_reviewer_search(by, values, first_call=True):
             }
         ],
     },
+    {
+        "name": "CrowdStrike Devices",
+        "macro_configurations": [
+            {
+                "macro_name": "cs_crowdstrike_devices",
+                "label": "CrowdStrike Devices Data",
+                "search_by": "sourcetype",
+                "search_values": "crowdstrike:device:json",
```

Comment:
It seems this sourcetype has more info of devices compared to CrowdStrike:Event:Streams:JSON